mockwebserver-deprecated/src/main/kotlin/okhttp3/mockwebserver/Dispatcher.kt

```diff
@@ -19,6 +19,8 @@ abstract class Dispatcher {
   @Throws(InterruptedException::class)
   abstract fun dispatch(request: RecordedRequest): MockResponse
 
+  abstract fun clear()
```

Comment:
What about an open method with an empty body here?

Reply:
I think this likely only makes sense on QueueDispatcher . . . could we just require callers to downcast to that type? I’d rather the callers downcast than have a method that doesn’t make sense on another dispatcher type.

Reply:
Fine for me, as long as it is documented so that users know about it.

Reply:
I reduced the Merge Request to just an `open fun clear()` in `QueueDispatcher`.